src/screens/SUE/SueReportScreen.tsx

```diff
@@ -393,7 +369,18 @@ export const SueReportScreen = ({
         if (!getValues(INPUT_KEYS.SUE.TITLE) || !getValues(INPUT_KEYS.SUE.TITLE).trim()) {
           return texts.sue.report.alerts.missingAnyInput;
         } else if (getValues(INPUT_KEYS.SUE.IMAGES)) {
-          const images = JSON.parse(getValues(INPUT_KEYS.SUE.IMAGES));
+          let images;
+
+          try {
+            images = JSON.parse(getValues(INPUT_KEYS.SUE.IMAGES));
+          } catch (error) {
+            console.error('Invalid JSON in images:', error);
+            return;
+          }
+
+          if (!images.length) {
```

Comment:
`undefined.length` will crash. please add a ? or make images a [] by default

Reply:
`images` has the default value `'[]'`

```
...
  } = useForm({
    mode: 'onBlur',
    defaultValues: {
      ...
      [INPUT_KEYS.SUE.IMAGES]: '[]',
      ...
    }
  });
```

Reply:
please check few lines before...`let images;`...which is undefined initially

Reply:
If the `images` value is `undefined`, we may get an error here. You are right. I will add a check.